pipeline-examples/checkout-to-custom-directory/getBuildCause.groovy

```diff
@@ -0,0 +1 @@
+checkout scm: [$class: 'GitSCM', branches: [[name: '*/master']], userRemoteConfigs: [[url: 'https://github.com/jglick/simple-maven-project-with-tests']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'bla']]]
```

Comment:
As this only works for Git, it might be a good idea to add `git` to the directory name this example is in.